### ISSUE TITLE
Uploads on Heroku are not persistent; use a third party service

### ISSUE THREAD

**Issue by therealklanni** [therealklanni/slack-invite-request#6]
I wasn't considering the fact that Heroku is instanced and any files will be lost whenever it spins down an instance to "sleep". We need uploads to be stored with an external service. We can potentially make this a part of the future Redis support, or create a plugin for uploading the image to any number of available third-party services such as Imgur.

In the mean time, as long as the uploads are viewed immediately it will work.

Any ideas or discussion on this topic is welcomed.

**Comment by pklapperich** [comment]
imagebin.ca maybe? They have a simple API http://imagebin.ca/tools.php and let you set expiration times, password protection, etc. Uploaded Images aren't easily browsable. Test with curl worked. After uploading it returned:
status:24upOaKwa25Q
url:http://ibin.co/24upOaKwa25Q

Another option is imgur; they also have an API and probably have a faster site. But the API requires OAuth login and you have to make sure to set the privacy options on your account: https://help.imgur.com/hc/en-us/articles/201746817-Image-and-album-privacy-explained-

**Comment by therealklanni** [comment]
Yeah, I was looking at Imgur. Haven't tested it out yet, been too busy with other stuff. I'll look into imagebin as well, that might be a better option.
